src/workloads/execution/UnshardedLookup.yml

```diff
@@ -71,6 +95,38 @@ Actors:
             }}
           ]
         cursor: {batchSize: *batchSize}
+    - OperationMetricsName: ShardedSourceLookup1
```

Comment:
I should give these better names. This benchmark needs to scan all shards of the left(input) collection. The other test however only needs to scan one since it matches on the shard key.